### PR TITLE
 Implement LocalRegistry for testing

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

File: src/registry/test-data/test-api-0.1.0.tgz
Comment (mara214) [inline src/registry/test-data/test-api-0.1.0.tgz:1]: Can we store the test data in the `tests/*` dirs eventually?

**Review by mara214 — COMMENTED** [review]
Few nits but generally it lgtm!

File: src/registry/test-data/test-api-0.1.0.tgz
Comment (uschi2000) [reply src/registry/test-data/test-api-0.1.0.tgz:1]: done

File: src/registry/mod.rs [hunk 1]
```diff
@@ -3,6 +3,8 @@
 use crate::{manifest::Dependency, package::Package};
 
 mod artifactory;
+#[cfg(test)]
```
Comment (uschi2000) [inline src/registry/mod.rs:6]: fyi @mara214 , I am now hiding the whole module in the test config

**Review by mara214 — APPROVED** [review]

**Pull request merged**